HW02_B_Mimic_starter.Rmd

```diff
@@ -59,9 +59,12 @@ data("iris")
 Using the iris dataset, make this graph:
 ```{r graph 2 code, echo=FALSE}
 
+ggplot(iris, mapping = aes(x = Sepal.Length, y = Petal.Length, color = Species, shape =Species)) + 
```

Comment:
It looks like ordering the facets is the only thing you struggled with for this graph -- everything else looks great! I did this by making the "correct" order its own factor, like this: iris$SpeciesOrdered <- factor(iris$Species, levels = c("versicolor", "setosa","virginica"))

Once I did that, the shape and color orders were messed up, so I did this too:
  scale_color_manual(values = c("green","red","dodgerblue2")) + #specify colors
  scale_shape_manual(values = c(17, 16, 15)) + #specify shapes

This was a really tough part of this graph -- it's nice to know I wasn't the only one struggling with it